common/lib/xmodule/xmodule/modulestore/tests/test_mixed_modulestore.py

```diff
@@ -454,6 +454,10 @@ def test_course_version_on_block(self, default_ms):
         self.assertEquals(len(blocks), 7)
         for block in blocks:
             self.assertEquals(block.course_version, course_version)
+            # ensure that when the block is retrieved from the runtime cache,
+            # the course version is still present
+            cached_block = course.runtime.load_item(block.location)
+            self.assertEquals(cached_block.course_version, block.course_version)
```

Comment:
Prefer assertEqual to assertEquals.  The later has been romoved in python 3.  R*e*moved.